### DISCUSSION TITLE
paragraphs in metadata

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1517] [Q&A]
could you help, please, how can I create multiple paragraphs in metadata?
for example, in my metadata for 3.4.4 I want to create 3 paragraphs in the REC_USE_LIM section

**Comment by christoph-LSN** [comment]
@ukraine-sdg perhaps you can try to use 

Text
`<br>` paragraph
Text

**Comment by LucyGwilliamAdmin** [comment]
@ukraine-sdg the easiest way to edit the metadata may be to make use of the metadata forms.

You can access these by:
1. go to an indicator page on the **staging** site e.g. https://ukraine-sdg.github.io/site/3-4-4/
2. scroll down to the Metadata section
3. click the Edit tab
4. Click Edit Metadata

Once you get to a form for an indicator, fill in the fields you want and then scroll back to the top and click Download configuration. Then click Go to repository, and upload the file that you just downloaded.

Note: Underneath each field you will see Scope and Field, these relate to the tab on which the inputted information you want to show so if working on national metadata you will need to scroll to about half way down the page

More info about the Metadata format can be found in read the docs: https://open-sdg.readthedocs.io/en/latest/metadata-format/
